### ISSUE TITLE
[BUG] Checklists not working with mock-user login (-1)

### ISSUE THREAD

**Issue by BrinkleyPT** [V1P3R-FMG/free-map-genie#62]
**Describe the bug**
When I go to the game main page where all the maps are listed and click on the checklist link, it says I'm not logged in and cannot tick any checkbox or checklist/ticking box.

**To Reproduce**
1. Go to 'Black Myth: Wukong or Elden Ring main/index page (where all map names/links are listed)'
2. Click on 'BOSS CHECKLIST'
3. Scroll down to 'checkbox/ticking boxes section' and try to mark/tick any box
4. See error "Login to track progress"

**Expected behavior**
Expected to be able to check all the boxes I wanted to track progress on.

**Screenshots**
![Screenshot_2024-11-30-00-04-40-896_com kiwibrowser browser](https://github.com/user-attachments/assets/1ad24e99-26e2-4790-be5b-6d30e30f83ca)
![Screenshot_2024-11-30-00-04-34-879_com kiwibrowser browser](https://github.com/user-attachments/assets/2604c806-35e2-4677-a87c-f17fd07fa87f)
![Screenshot_2024-11-30-00-04-24-787_com kiwibrowser browser](https://github.com/user-attachments/assets/e32d33eb-6fdb-494e-ab1b-de04be14939f)
![Screenshot_2024-11-30-00-04-11-629_com kiwibrowser browser](https://github.com/user-attachments/assets/2b908d7c-6a72-4ac3-ab21-e3834927fd49)
![Screenshot_2024-11-30-00-03-44-066_com kiwibrowser browser](https://github.com/user-attachments/assets/33724cad-81d3-4d0c-9243-91a71bde13ff)

**Desktop (please complete the following information):**
 - OS: Android 13
 - Browser: Kiwi Browser
 - Version: 124.0.6327.4

**Additional context**
Perhaps this won't matter that much right now, but it would be nice to have it working on V3 or so.

**Comment by MrFusiion** [comment]
yeah working on and off on V3 when I find time, but once its released checklists should be working again.
